### DISCUSSION TITLE
How to open the modal for search files that is found in Upload field

### DISCUSSION THREAD

**Discussion by thewebartisan7** [orchidsoftware/platform#2032] [Q&A] [answered]
I found that there is a hidden modal for search files in Upload field, but no way to open it.

<img width="983" alt="Screenshot 2021-12-27 at 13 05 30" src="https://user-images.githubusercontent.com/57477934/147472421-33694f4e-3acd-451a-96c8-dafec9b2fd50.png">

**Comment by tabuna** [comment, marked as answer]
Hi, this is the media directory of previously uploaded files, searchable by name. You can use it like this:

```php
Upload::make('...')->media()
```

**Reply by thewebartisan7** [reply]
Thanks that worked fine. I think would be good to make this a bit more clear in docs. Maybe I missed this, but I carefully read all docs more than once and didn't find it. It should be here https://orchid.software/en/docs/field/#upload
